src/scripts/towns/TownList.ts

```diff
@@ -323,6 +323,33 @@ const CelebiProfIvy = new NPC ('Prof. Ivy', [
 ],
 { requirement: new QuestLineStepCompletedRequirement('Unfinished Business', 2, GameConstants.AchievementOption.less) });
 
+const Informant1 = new NPC('Informant', [
+    'In a shady warehouse, you find the informant. He is a Mr. Mime, and he doesn\'t seem willing to divulge the information you need.',
+],
+{ requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('Detective Pikachu', 2), new QuestLineStepCompletedRequirement('Detective Pikachu', 4, GameConstants.AchievementOption.less )]) });
+const Informant2 = new NPC('Informant', [
+    'The Mr. Mime signals to you that this is an illicit drug called R, and that it is frequently used in the underground fighting rings near the Battle Frontier',
+],
+{ requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('Detective Pikachu', 4), new QuestLineStepCompletedRequirement('Detective Pikachu', 6, GameConstants.AchievementOption.less )])});
+
+const MewTwo1 = new NPC('Mewtwo', [
+    'You were wise to seek my out. Howard Clifford has been decieving you. He is the one making the R drug, and....',
```

Comment:
Couple spelling errors.
'You were wise to seek me out. Howard Clifford has been deceiving you. He is the one making the R drug, and...',